docs/retention_policy.md

```diff
@@ -0,0 +1,101 @@
+Retention Policies
+==================
+
+A retention policy blocks permanent deletion of content for a specified amount of time. Admins can create retention policies and then later assign them to specific folders or their entire enterprise.
+
+Create Retention Policy
+-----------------------
+
+You can call `client.create_retention_policy(policy_name, disposition_action)` to let you create a new indefinite retention policy.
```

Comment:
These method signatures should have all parameters (including optional ones with default values) displayed